pkg/virt-controller/watch/vmi_test.go

```diff
@@ -851,7 +851,7 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 
 			controller.Execute()
 
-			Expect(pod.Annotations).To(HaveKey(downwardapi.NetworkInfoAnnot))
+			expectPodAnnotations(pod, HaveKey(downwardapi.NetworkInfoAnnot))
```

Comment:
:thinking: Were we just confirming that the pod cache was corrupted?!?!
+1

Reply:
Yes we were! :face_holding_back_tears: